test/helpers/components/operator/installation.go

```diff
@@ -70,6 +83,11 @@ func InstallFromGithub(releaseTag string, withCsi bool) features.Func {
 	return manifests.InstallFromUrls(manifestsUrls)
 }
 
+func UninstallFromGithub(releaseTag string, withCsi bool) features.Func {
```

Comment:
Function is not used, UninstallOperatorFromRelease() uses this function but is never called anywhere else